models/User.js

```diff
@@ -51,11 +51,21 @@ module.exports = function(sequelize, DataTypes) {
     return devices.map(d => d.id);
   };
 
+  const getFromId = async function(id) {
+    return await this.findById(id);
+  };
+
+  const getFromName = async function(name) {
```

Comment:
Just curious, how is `const foo = async function(...` different from `async function foo(...`?

Reply:
With `async function foo(...` you can reassign foo to something else, `foo = 1` and with `const` you can't.